rpcs3/Emu/System.h

```diff
@@ -366,7 +367,7 @@ class Emulator final
 
 	bool IsRunning() const { return m_state == system_state::running; }
 	bool IsPaused()  const { return m_state >= system_state::paused; } // ready/starting are also considered paused by this function
-	bool IsStopped() const { return m_state <= system_state::stopping; }
+	bool IsStopped(bool test_fully = false) const { return test_fully ? m_state == system_state::stopped : m_state <= system_state::stopping; }
```

Comment:
This bug we need to fix in many places in GUI I think, but next pr. (we are testing as if m_state == system_state::stopped)